tools/value-generation/README.md

```diff
@@ -3,11 +3,10 @@
 This directory contains simple Python tooling to auto-generate a `values.yaml` file with comments from a CRD.
 
 `docker build -t values-generator .`
-`docker run values-generator > values.yaml`
 
 The YAML is output to standard output of the container.
 To run with a custom CRD:
-`docker run values-generator - < <CRD>`
+`docker run values-generator -f full - < <CRD>`
```

Comment:
I can't see any changes to handle this flag - did we miss it before?

Reply:
Right it's set to "full" by default, so having it explicit could document that it's being used